routers/auth.py

```
@@ -1,10 +1,130 @@
+        "message": "로그인에 성공하였습니다.",
+        "data": {
+            "userId": db_user["userId"],
+            "nickname": db_user["nickname"],
+            "accessToken": access_token,
+            "tokenType": "Bearer",
+            "expiresIn": 60 * 60
+        }
+    }
+
+@router.post("/tokens/refresh")
+async def refresh_access_token(request: Request, response: Response):
```

Ruff (0.14.13)

Unused function argument: `response`

(ARG001)